src/app/page.tsx

```diff
@@ -60,7 +61,11 @@ const HomePage = () => {
             className="text-primary flex items-center gap-1 transition hover:underline duration-1000"
```

Comment:
important: fix height of `About the data portal`. The background is cut when the text is too short.

<img width="1588" alt="image" src="https://github.com/user-attachments/assets/a29b5be0-2e85-4b71-ba45-51dfcb55eb8b">

Reply:
min 300 is set.